src/components/Reports/PartnerGivingAnalysisReport/Table/Table.tsx

```diff
@@ -126,12 +123,18 @@ export const PartnerGivingAnalysisReportTable: FC<
                   />
                 </TableCell>
                 <TableCell>
-                  <ContactName
-                    onClick={() => onClick(contact.id)}
-                    onMouseEnter={preloadContactsRightPanel}
-                  >
-                    {contact.name}
-                  </ContactName>
+                  <NextLink href={contactUrl} passHref>
+                    <Link
```

Comment:
These links are also always underlined now instead of underlined on hover. Also, is the `component="a"` necessary? I think the default component for `Link`s is an `<a>` tag.